tests/sqllogictests/suites/base/15_procedure/15_0002_procedure.test

```diff
@@ -49,17 +44,27 @@ call procedure p1();
 ----
 2
 
+query T
+call procedure p1(0, 0);
+----
+6
+
+query T
+call procedure p1(1,10);
+----
+20
+
 query T
 select name, arguments from system.procedures where name = 'p1';
 ----
 p1 p1() RETURN (Int32)
-p1 p1(Int32,Int32) RETURN (Int32)
+p1 p1(uint8,uint8) RETURN (Int32)
```

Comment:
lower case ?

Reply:
Yes. In this pr I make the type name as lower case when parse.

https://github.com/datafuselabs/databend/pull/16453/files#diff-391b369d101e47b8108e9a90993e8a08f926edd73ddc5a2d0c2bed1a4bf90703R2079-R2082
https://github.com/datafuselabs/databend/pull/16453/files#diff-391b369d101e47b8108e9a90993e8a08f926edd73ddc5a2d0c2bed1a4bf90703R2149-R2150

e.g.

create procedure p1(x UInt8) -> should can be drop procedure p1(uint8)

Reply:
Done